Application/EdFi.Ods.Api/Security/Authorization/AuthorizationBasisMetadataSelector.cs

```diff
@@ -305,8 +294,6 @@ int GetBitValuesByAction(string action)
 
     private class ClaimCheckResponse
     {
-        public bool Success { get; set; }
```

Comment:
Removed this property because after these changes, all possible failures throw an exception, so it became redundant.

Reply:
I reverted to returning a result since this path already supported it. I just added a couple of properties to support specifying the problem detail and type parts.